src/models/unified_api_server.py

```
@@ -0,0 +1,443 @@
+                "gpu_allocated": torch.cuda.memory_allocated() / 1024**3,
+                "gpu_reserved": torch.cuda.memory_reserved() / 1024**3,
+            }
+
+        return HealthResponse(
+            status="healthy" if all(models_loaded.values()) else "degraded",
+            timestamp=datetime.now(),
+            models_loaded=models_loaded,
+            memory_usage=memory_usage
+        )
+
+    def run(self, host: str = "0.0.0.0", port: int = 8000):
```

Ruff (0.12.2)

Possible binding to all interfaces

(S104)